taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/exception/MismatchedInputExceptionMapper.java

```diff
@@ -0,0 +1,14 @@
+package pro.taskana.adapter.camunda.outbox.rest.exception;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+import spinjar.com.fasterxml.jackson.databind.exc.MismatchedInputException;
+
+public class MismatchedInputExceptionMapper implements ExceptionMapper<MismatchedInputException> {
```

Comment:
Where do we use this exception?

Reply:
If there is no body at all, this exception is thrown from jackson...I thought it is better to let the user know what went wrong instead of a generic 500 error in that case

Reply:
perfect.